samples/logging-sample/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import ("context"
```

Comment:
NIt: weird formatting here?

Reply:
Also, FMU the standard is separate stdlib and non-stdlib imports into two different chunks.

Reply:
Updated - LMK if it needs more editing or if there is a style guide I can refer to?